spec/geo_ruby/gpx_spec.rb

```diff
@@ -34,6 +34,11 @@
       expect(@gpxfile[0].y).to be_within(0.0001).of(48.731813)
     end
 
+    it 'should point last record to last waypoint' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.